f2/server.js

```diff
@@ -4,6 +4,13 @@ const path = require('path')
 const formidable = require('formidable')
 const { getAllCerts, encryptAndSend } = require('./src/utils/encryptedEmail')
 const { selfHarmWordsScan } = require('./utils/selfHarmWordsScan')
+var clamd = require('clamdjs')
+var fs = require('fs')
+var scanner = clamd.createScanner(
+  'mpsccdscybercrimeclamav.canadacentral.azurecontainer.io',
```

Comment:
If we move this `scanner =...` stuff to below the `require('dotenv').config()` then we can use an env var for the url, i.e. something like `process.env.CLAM_URL`

Reply:
done